test/unittest/compiler/unittest_interpreter.cpp

```diff
@@ -294,3 +297,110 @@ GTEST_PARAMETER_TEST(nntrainerAutoInterpreterTest, nntrainerInterpreterTest,
   mkTc(makeGraph({fc0, flatten}), "simple_fc_backbone.ini", ini_interpreter)
 ));
 // clang-format on
+
+/**
+ * @brief Fully Connected Layer weights transpose(NCHW -> NHWC) unittest
+ *
+ */
+
+TEST(nntrainerInterpreterTflite, simple_flatten) {
```

Comment:
I'm not sure, but can we extract common part of this to the parameterized tests (line 294) ??

```
GTEST_PARAMETER_TEST(nntrainerAutoInterpreterTest, nntrainerInterpreterTest,
                        ::testing::Values(
  mkTc(makeGraph({fc0, flatten}), "simple_fc.ini", ini_interpreter),
  mkTc(makeGraph({fc0, flatten}), "simple_fc_backbone.ini", ini_interpreter)
  mkTc(makeGraph({fc0, flatten}), "simple_fc.ini", tflite_interpreter),
  mkTc(makeGraph({fc0, flatten}), "simple_fc_backbone.ini", tflite_interpreter)
));
```

Reply:
OK.